### DISCUSSION TITLE
Account Recovery Query - Urgent

### DISCUSSION THREAD

**Discussion by AquaOwO** [community/community#111303] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I'm very sorry for posting here, but my account "**dreamjz**" has been **suspended without any notification email** for a week now, and I'm feeling extremely urgent and anxious because:

1. My resume and job applications reference my GitHub account profile, which is now unavailable.
2. I have contributed to some open-source projects, but I'm currently unable to submit pull requests and contact maintainers.

I have create a Ticket (Ticket ID: 2652249 ) for a week and didn't get any response now.

I'm not sure if there are any additional steps I need to take to restore my account. If there are any, please let me know. 

**Additional information**

On the day my account was suspended, as well as in the preceding weeks, I received a large number of phishing emails from unknown repositories. I'm not sure if this is the reason for the suspension.

![pic1](https://github.com/community/community/assets/153427736/13cf254d-c5dd-474a-9d4e-0db9d88aebd2)

Thank you

**Comment by Arjun650** [comment]
I understand your urgency and frustration with the suspension of your GitHub account. Given the potential security concerns, it's vital to follow up on your existing ticket (Ticket ID: 2652249) with GitHub support promptly. In the meantime, consider explaining the situation in your job applications and providing alternative contact information. Additionally, try reaching out to GitHub support through multiple channels, such as their official Twitter account or community forums, for expedited assistance.
😌

**Comment by SanjaRogovic** [comment]
Here is the link to GitHub official account recovery policy: https://docs.github.com/en/site-policy/other-site-policies/github-account-recovery-policy

I believe this note could be important to you: "If you cannot use any recovery methods, you have permanently lost access to your account. However, you can unlink an email address tied to the locked account. The unlinked email address can then be linked to a new or existing account." Link for more info: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/unlinking-your-email-address-from-a-locked-account

Additionally,  have you tried contacting support? You can do this here: https://support.github.com/

Hope you find solution fast :) 

Sanja

**Comment by queenofcorgis** [comment, marked as answer]
Hi @AquaOwO 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
